packages/govern-console/src/containers/ProposalDetails/ProposalDetails.tsx

```diff
@@ -293,7 +293,13 @@ const ProposalDetails: React.FC<ProposalDetailsProps> = ({ onClickBack }) => {
   }, [networkName]);
 
   const proposalInstance = React.useMemo(() => {
-    if (provider && account && daoDetails && proposalInfo) {
+    if (
+      provider &&
+      daoDetails &&
+      proposalInfo &&
+      account.address &&
```

Comment:
any specific reason why writing account.address && account.signer was needed instead of account directly ?